tests/jetty-test-util/pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.jetty.tests</groupId>
+    <artifactId>tests</artifactId>
+    <version>12.1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>jetty-test-util</artifactId>
```

Comment:
What is this for?

Reply:
This new module seems to be very OpenID specific.
Why is it called `jetty-test-util` ?

Reply:
It's so we can share the `OpenIdProvider` instead of having like 5 different copies of it.

I don't mind if we call it `jetty-openid-test-util` or whatever, but it has no openid specific dependency so no reason it couldn't have non-openid utilities eventually. It just needs the dependency on `jetty-server`.

Reply:
I think `jetty-test-common` is a better name

Reply:
we could definitely share some code here from all the test-eeXX-distribution.

Reply:
Renamed to `jetty-test-common`.